php/index.php

```diff
@@ -119,230 +158,266 @@ function success($status, $result, $contenttype = 'application/json')
+            return $helper->success($res, 200, $result);
         } else {
-            $app->resource_error(500, 'update statement went wrong');
+            return $helper->resource_error($res, 500, 'update statement went wrong');
         }
     } else {
-        $app->not_found("Post with id=\"{$id}\" not found");
+        return $helper->not_found($res, "Post with id=\"{$id}\" not found");
     }
-})->name('post-like-put');
+})->setName('post-like-put');
 
 /**
  * Get a list of all posts.
  */
-$app->get('/posts', function() use ($app)
+$app->get('/posts', function (Request $req, Response $res, $args = [])
 {
-    $qry = $app->conn->prepare("SELECT p.*, c.name as country, c.geometry as area
+    $helper = $this->get('helper');
+    $qry = $helper->conn->prepare("SELECT p.*, c.name as country, c.geometry as area
             FROM guestbook.posts AS p, guestbook.countries AS c
             WHERE within(p.\"user\"['location'], c.geometry)
             ORDER BY p.created DESC");
     $qry->execute();
     $result = $qry->fetchAll(PDO::FETCH_ASSOC);
-    $app->success(200, $result);
-})->name('posts-get');
+    return $helper->success($res, 200, $result);
+})->setName('posts-get');
 
 /**
- * returns all images that are saved in the crate blob store.
+ * Return all images that are saved in the CrateDB blob store.
  */
-$app->get('/images', function() use ($app)
+$app->get('/images', function (Request $req, Response $res, $args = [])
 {
-    $qry = $app->conn->prepare("SELECT digest, last_modified FROM blob.guestbook_images");
+    $helper = $this->get('helper');
+    $qry = $helper->conn->prepare("SELECT digest, last_modified FROM blob.guestbook_images");
     $qry->execute();
     $result = $qry->fetchAll(PDO::FETCH_ASSOC);
-    $app->success(200, $result);
-})->name('images-get');
+    return $helper->success($res, 200, $result);
+})->setName('images-get');
 
 /**
- * inserts an image
+ * Insert an image.
  */
-$app->post('/images', function() use ($app)
+$app->post('/images', function (Request $req, Response $res, $args = [])
 {
-    $data = json_decode($app->request->getBody());
-    if (!isset($data->blob)) {
-        $app->argument_required('Argument "blob" is required');
-        return;
+    $helper = $this->get('helper');
+    $data = $req->getParsedBody();
+    if (!isset($data["blob"])) {
+        return $helper->argument_required($res, 'Argument "blob" is required');
     }
-    $content = base64_decode($data->blob);
+    $content = base64_decode($data["blob"]);
     $digest  = sha1($content);
-    $ch      = curl_init("{$app->config['blob_url']}guestbook_images/{$digest}");
+    $ch      = curl_init("{$helper->config['blob_url']}guestbook_images/{$digest}");
     curl_setopt($ch, CURLOPT_CUSTOMREQUEST, "PUT");
     curl_setopt($ch, CURLOPT_POSTFIELDS, $content);
     curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
     $result = curl_exec($ch);
     $info   = curl_getinfo($ch);
+    //print("INFO");
+    //print_r($info);
```

Comment:
Remove these lines?

Reply:
Thanks for spotting. 39b07742a removes the lines.